app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+  <%= f.label :name %><br />
+  <%= f.text_field :name %>
+</div>
+
+<div class="field">
+  <%= f.label :bio %><br />
+  <%= f.text_field :bio %>
+</div>
+
+<div class="field">
+  <%= f.label :photo %><br />
+  <%= f.text_field :photo %>
+</div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Kindly improve the design of your form to match the wireframe. The form should be somewhat centered and wider

  your sign up form | form according to design
  :---:|:---:
  ![ruiw](https://user-images.githubusercontent.com/36542986/216268014-7dc8f1fd-3a75-4942-842e-46ec2b612ec3.jpg) | ![rwo](https://user-images.githubusercontent.com/36542986/216268018-3b6dd313-a8af-459a-bd43-4031ec8f9aaa.jpg)